code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -424,6 +434,11 @@
 			P.forceMove(C)
 			AP.Grant(C)
 			break
+	for(var/obj/item/assembly/signaler/S in src)
+		for(var/datum/action/item_action/hands_free/activate_signaler/AS in S.actions)
+			S.forceMove(C)
+			AS.Grant(C)
```

Comment:
Use more informative var names please.

Reply:
This I was just following the form of the existing one with the pill, thought maybe there was a reason for things like this being 1 or 2 characters. I can run through and change them all to be more verbose.